edb/load_ext/main.py

```diff
@@ -104,6 +104,32 @@ def get_pg_config(pg_config_path: pathlib.Path) -> dict[str, str]:
     return config
 
 
+def install_edgedb_extension(
+    pkg: pathlib.Path,
+    ext_dir: pathlib.Path,
+) -> None:
+    target = ext_dir / pkg.stem
+    print("Installing", target)
+
+    shutil.rmtree(target, ignore_errors=True)
```

Comment:
I'm a bit uneasy about nuking first.  Let's install into a tempdir and then do a rename followed by rmtree.

Reply:
No `rmtree` needed then, right?

Reply:
You still want to remove the old thing no?

```
unpack -> tmpdir
mv ext ext~
mv tmpdir ext
rm -rf ext~
```